env/env.mock.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
     host: 'localhost',
-    port: 9000
+    port: 3000
```

Comment:
why do we change config files in scope of task implementation?
(actually that happens second time :suspect: )